src/install.rs

```diff
@@ -233,6 +236,53 @@ fn write_platform(mountpoint: &Path, platform: &str) -> Result<()> {
     Ok(())
 }
 
+/// Copy networking config if asked to do so
+fn copy_network_config(mountpoint: &Path, net_config_src: &str) -> Result<()> {
+    eprintln!("Copying networking configuration from {}", net_config_src);
```

Comment:
Since this isn't an error, let's use `println!`.

Reply:
will look into this. Thanks for the suggestion.

Reply:
Nope, all of our existing status logging goes to stderr.

Reply:
So I'll leave this as is in this PR, but it's probably worth us having a followup where we convert our non-error status logging to non stderr, right?

Reply:
Why?  Status reports aren't the primary output of the program, so it makes sense to me that they would go to stderr.

Reply:
I grant you this is somewhat debatable but I think precedent weighs most strongly towards `stderr` is for *errors*.

Reply:
Yes I typically think that you only print to stderr if there is a problem. That's mostly what I've experienced. I don't know if there is a written down standard anywhere.

Reply:
I don't agree.  If we ever add e.g. a JSON report describing what actions `install` has taken, that would go to stdout for piping to another process, and the status reports would need to not be commingled with it.

@lucab, thoughts?

Reply:
> I don't agree. If we ever add e.g. a JSON report describing what actions `install` has taken, that would go to stdout for piping to another process, and the status reports would need to not be commingled with it.

Sounds like a good use case for a `--quiet`.

Reply:
Either way maybe we should move this coversation into an issue since it no longer really reflects a change we are going to make in this PR.

Reply:
follow up: https://github.com/coreos/coreos-installer/issues/214

Reply:
> I don't agree. If we ever add e.g. a JSON report describing what actions install has taken, that would go to stdout for piping to another process, and the status reports would need to not be commingled with it.

What you're talking about is one of the most long-standing things in how Unix pipelines work - the fact they're just connected by bytestreams and there's no common APIs or structure.  A lot of successor projects try to address this, from [Windows Powershell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/getting-started-with-windows-powershell?view=powershell-7) to [nushell](https://github.com/nushell/nushell) to many others.  (In fact long ago I tried to write one myself...)

Practically speaking, I think it's better to do e.g. `--status-report-json /path/to/status.json`.  Or alternatively `--output=json` and that also suppresses internal printing.